Liquidity-provider-server automated update of documentation update-from-lps-20250519-141704 (#325)
rsksmart/devportal: rif/rns/libs/javascript/Operations.md

#### `setOwner`

Sets a subdomain owner, it does not check for subdomain availability.

> It submits a [setSubnodeOwner](/rif/rns/architecture/registry#set-a-subdomain) transaction, so if the subdomain exists, it will overwrite the existing owner and will set the resolver of the parent domain.  If the subdomain does not exists, will create it.

Reviewer: ```suggestion
It submits a [`setSubnodeOwner`](/rif/rns/architecture/registry#set-a-subdomain) transaction, so if the subdomain exists, it will overwrite the existing owner and will set the resolver of the parent domain.  If the subdomain does not exists, will create it.
```

#### `create`

Creates a new subdomain under the given domain tree if it is available, and sets its resolution if `addr` is provided.

> It could send one, two or three transactions based on the value of the sent parameters.

Reviewer: ```suggestion
It may send one, two, or three transactions, based on the value of the sent parameters.
```

- `owner`: The owner of the new subdomain. If not provided, the address who executes the tx will be the owner
- `addr`: The address to be set as resolution of the new subdomain

> > If `addr` is not provided, no resolution will be set and will send only one [setSubnodeOwner](/rif/rns/architecture/registry#set-a-subdomain) transaction.

Reviewer: ```suggestion
> If `addr` is not provided, no resolution will be set, and will send only one [`setSubnodeOwner`](/rif/rns/architecture/registry#set-a-subdomain) transaction.
```

>

> > If `owner` is not provided, the sender will be set as the new owner, will send one [setSubnodeOwner](/rif/rns/architecture/registry#set-a-subdomain) transaction

Reviewer: ```suggestion
> If `owner` is not provided, the sender will be set as the new owner, and will send one [`setSubnodeOwner`](/rif/rns/architecture/registry#set-a-subdomain) transaction.
```

>

> > If `owner` and `addr` are provided and `owner` is equals to the sender, will send two transactions: [setSubnodeOwner](/rif/rns/architecture/registry#set-a-subdomain) and [addr](/rif/rns/architecture/Resolver#addr).

Reviewer: ```suggestion
> If `owner` and `addr` are provided and `owner` is equals to the sender, will send two transactions: [`setSubnodeOwner`](/rif/rns/architecture/registry#set-a-subdomain), and [`addr`](/rif/rns/architecture/Resolver#addr).
```

>

> > If `owner` and `addr` are provided but `owner` is different from the sender, then three transactions will be sent: [setSubnodeOwner](/rif/rns/architecture/registry#set-a-subdomain), [addr](/rif/rns/architecture/Resolver#addr) and [setSubnodeOwner](/rif/rns/architecture/registry#set-a-subdomain) again.

Reviewer: ```suggestion
> If `owner` and `addr` are provided, but `owner` is different from the sender, will send three transactions: [`setSubnodeOwner`](/rif/rns/architecture/registry#set-a-subdomain), [`addr`](/rif/rns/architecture/Resolver#addr), and [`setSubnodeOwner`](/rif/rns/architecture/registry#set-a-subdomain) again.
```
